pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -220,6 +225,32 @@ var _ = Describe("Clone", func() {
 		})
 	}
 
+	expectCloneUpdateWithSnapshotName := func(phase clonev1alpha1.VirtualMachineClonePhase, snapshotName string) {
+		client.Fake.PrependReactor("update", clone.ResourceVMClonePlural, func(action testing.Action) (handled bool, ret runtime.Object, err error) {
+			update, ok := action.(testing.UpdateAction)
+			Expect(ok).To(BeTrue())
+
+			vmClone := update.GetObject().(*clonev1alpha1.VirtualMachineClone)
```

Comment:
Check this cast too, as above?

Reply:
*Type assertion
https://go.dev/tour/methods/15

Reply:
Done. However, could you please explain the reason for needing a check at this point?